views/pages/Search-RestUser.ejs

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+  
+</head>
+
```

Comment:
There is no nav in this page, how are the users going to navigate to and from this page?